qa/rpc-tests/sc_async_proof_verifier.py

```diff
@@ -173,7 +173,7 @@ def run_test(self):
 
         try:
             # The send_certificate call must be ok since the proof verification is disabled on node 2
```

Comment:
You forgot to update the comment ->"sc_send_certificate"

Reply:
Fixed.